client/lib/site/index.js

```diff
@@ -50,11 +50,13 @@ Emitter( Site.prototype );
 
 Site.prototype.attributes = function( attributes ) {
 	attributes = attributes || {};
-
-	for ( var prop in attributes ) {
+	for ( const prop in attributes ) {
 		if ( attributes.hasOwnProperty( prop ) ) {
 			this[ prop ] = attributes[ prop ];
 		}
+		if ( 'updates' === prop ) {
+			this.update = attributes[ prop ];
```

Comment:
I think it would be cleaner to move whenever we use `this.update` to start using `this.updates`.
What do you think?